deployment/docker/rules/alerts-cluster.yml

```diff
@@ -34,7 +33,8 @@ groups:
           sum(vm_data_size_bytes) by(job, instance) /
           (
            sum(vm_free_disk_space_bytes) by(job, instance) +
-           sum(vm_data_size_bytes) by(job, instance)
+           sum(vm_data_size_bytes) by(job, instance) -
+           sum(vm_free_disk_space_limit_bytes) by(job, instance)
```

Comment:
[My question](https://github.com/VictoriaMetrics/helm-charts/pull/1814#issuecomment-2504328166) about change to disk related rules still remains.
`DiskRunsOutOfSpace` alert is expected to trigger when free disk space falls below 20%. With the proposed change, if the total disk size is `1000G` and `-storage.minFreeDiskSpaceBytes=200G`, the alert would be firing when VM uses only 640G, which is unreasonable as there would still be 36% disk space remaining.
I think, it's ok for this alert to become non-triggering if users set `-storage.minFreeDiskSpaceByte` bigger than 200G, as vmstorage won't run out of disk.  And it works correctly with small `-storage.minFreeDiskSpaceByte`.

Reply:
hmm, I don't see the use case for `NodeBecomeReadonlySoon` over `DiskRunsOutOfSpace` though. vmstorage nodes automatically switch to readonly mode when there is less than -storage.minFreeDiskSpaceBytes of free space. The scenarios are:
1. In a small setup: total disk is 1000G, if user follows the recommendation and set `-storage.minFreeDiskSpaceBytes=200G`, both `NodeBecomeReadonlySoon` and `DiskRunsOutOfSpace` trigger simultaneously when VM uses 800G, give duplicated alerts.
2. In a larger setup: total disk is 100T, users are unlikely to follow the recommendation to set -storage.minFreeDiskSpaceBytes to 20% of the total, instead opting for a more reasonable value like 500G. In this case, `DiskRunsOutOfSpace` already fires before `NodeBecomeReadonlySoon`.
btw, a rare case: sometimes we recommend user to use `-storage.minFreeDiskSpaceBytes` to temporarily make some vmstorage nodes readonly, allowing data to rebalance to newly added nodes. And in this case, `NodeBecomeReadonlySoon` is a false alert.

Reply:
> both NodeBecomeReadonlySoon and DiskRunsOutOfSpace trigger simultaneously when VM uses 800G

nope, NodeBecomeReadonlySoon will trigger sooner as it makes no sense to set the sam threshold for it and will allow user to perform some actions to keep it writable

Reply:
@Haleygo 
removed impact of `vm_free_disk_space_limit_bytes` from DiskRunsOutOfSpace alerts and added `NodeBecomesReadonlyIn3Days` to cover this case

Reply:
@hagen1778 wdyt?